### ISSUE TITLE
How does one remove records?

### ISSUE THREAD

**Issue by krzkrzkrz** [bububa/MongoHub-Mac#26]
Im on Mac osx snow leapard. How does one delete records?

I click on the "Remove" tab. and then clck on the remove button. It says Affected Rows: 11. Nothing happens to my records..... Is this a bug?

**Comment by kingawesome** [comment]
I noticed the same thing. At the top of the remove tab it says:

 db.[dbname].[collectionname].remove({});

If I hop in to the command line interface and run that command it doesn't delete any records, but if I do this instead it does:
 db.[collectionname].remove({});

But only after I have selected a db using "use [dbname]"

I don't know enough about mongohub or mongo itself to know exactly what's going on here, or if this is a platform specific issue. Hopefully this observation will help clear the issue up though.

**Comment by bububa** [comment]
fixed this bug in the next coming version.

**Comment by erskingardner** [comment]
Where is the next version, this bug is annoying me....

**Comment by codecowboy** [comment]
@bububa Is this still a bug? I can't see how to remove all records.

**Comment by jemrisin** [comment]
i still can't see how to remove records, has this issue been resolved?

**Comment by codecowboy** [comment]
@jemrisin suggest you open a new issue as I have not heard anything.
